tests/unit_tests/cli/test_repl.py

```
@@ -0,0 +1,328 @@
+        cli.available_subagents = {"gensql"}
+        cmd_type, cmd, args = cli._parse_command("/notasubagent do something")
+        assert cmd_type == CommandType.CHAT
+        assert cmd == ""
+
+    def test_sql_trailing_semicolon_stripped(self, cli):
+        """Trailing semicolon is stripped before parsing."""
+        with patch("datus.cli.repl.parse_sql_type") as mock_parse:
+            from datus.utils.constants import SQLType
+
+            mock_parse.return_value = SQLType.SELECT
+            cmd_type, cmd, args = cli._parse_command("SELECT 1;")
```

Ruff (0.15.6)

[warning] Unpacked variable `cmd` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)